asm/disk.asm

```diff
@@ -111,7 +111,7 @@ load_binary_laddr_hi = *+1
     lda #0		;0 = load to memory (no verify)
     jsr LOAD
     bcs .disk_io_error
-    jmp CLRCHN
+    rts
```

Comment:
I think the intent of this CLRCHN call was to switch back to use keyboard as input device. Is it wrong?

Reply:
LOADB is called by, LS and V. Both are using keyboard and screen.
  LOAD doesn't change the active devices.

Reply:
load_binary may be inlined too.